src/main/java/com/github/krzysiekjodlowski/ox/BoardPrinter.java

```diff
@@ -0,0 +1,31 @@
+package com.github.krzysiekjodlowski.ox;
+
+/**
+ * Generates String representation of the game board.
+ * @author Krzysztof Jodlowski
+ */
+class BoardPrinter {
+    /**
+     * @param board Board object
+     * @return string representation of the Board
+     */
+    String getBoardRepresentation(final Board board) {
+        int boardSize = board.getSize();
+        int boardCapacity = board.getBoardCapacity();
```

Comment:
Ok, I get why you want to have board printing separate from board itself but maybe instead of using getters from board, first validate data, then add board size as a constructor parameter, one line after another, to both Board and BoardPrinter?

Reply:
I agree with that:) Changed according to remark:)